x-pack/platform/plugins/shared/streams/server/lib/streams/state_management/streams/wired_stream.ts

```diff
@@ -729,11 +738,12 @@ export class WiredStream extends StreamActiveRecord<Streams.WiredStream.Definiti
       const ancestorFailureStore = ancestorStream.getFailureStore();
       if (!isInheritFailureStore(ancestorFailureStore)) {
         if (hasAncestorWithChangedFailureStore) {
+          const effectiveFailureStore = this.getInheritedFailureStoreFromAncestors(desiredState);
```

Comment:
we don't need this here since we already have the concrete `ancestorFailureStore` configuration